Makefile

```diff
@@ -30,16 +28,15 @@ distclean: clean
 
 .PHONY: clean
 clean:
-	@make -C rxtools clean
-	@make -C rxmode clean
-	@make -C brahma clean
-	@make -C theme clean
-	@make -C rxinstaller clean
-	@make $(CAKEFLAGS) -C CakeHax clean
+	@$(MAKE) -C rxtools clean
+	@$(MAKE) -C rxmode clean
+	@$(MAKE) -C brahma clean
+	@$(MAKE) -C theme clean
+	@$(MAKE) -C rxinstaller clean
+	@$(MAKE) $(CAKEFLAGS) -C CakeHax clean
 	@rm -f $(tools) payload.bin data.bin rxTools.dat
 
-.PHONY: release
-release: rxTools.dat rxtools/font.bin brahma/brahma.3dsx brahma/brahma.smdh theme doc rxinstaller.nds
+release: rxTools.dat rxtools/font.bin all-target-brahma all-target-theme rxinstaller.nds
 	@mkdir -p release/mset release/ninjhax
 	@cp rxTools.dat release
 	@cp rxtools/font.bin release/rxTools
```

Comment:
I do suggest anyone to change line `@mkdir -p release/mset release/ninjhax` to add `release/rxTools`.
Or else this `@cp rxtools/font.bin release/rxTools` would create a file and prevent building cleaned ones properly.